aeloggingstackdriver.go

```diff
@@ -203,6 +203,28 @@ func logFromContext(ctx context.Context, sev logtypepb.LogSeverity, format strin
 	}
 }
 
+func AddLoggingLabels(log Logging, labels map[string]string) error {
+	levelLogger, ok := log.(*LevelLogger)
+	if !ok {
+		return fmt.Errorf("failed to add log labels, log needs to be a level logger")
+	}
+	stackdriverLogger, ok := levelLogger.wrappedLogger.(stackdriverLogging)
+	if !ok {
+		return fmt.Errorf("failed to add log labels, wrapped log needs to be a stackdriver logger", )
+	}
+	ctxVal := (stackdriverLogger.c).Value(loggingCtxKey)
```

Comment:
Does it make sense to just have this as a member of stackdriverLogger if that's the only implementation to which this applies?

```
func (s *stackdriverLogger) AddLoggingLabels(labels map[string]string) {
}
```

Reply:
I originally debated doing that approach but then went a different way mainly because the caller of this is in appengine where we just pass around the generic Logging interface and it doesn't have access to cast down to stackdriverLogger. So on quick glance I was thinking to make that work I'd need to add the AddLoggingLabels function to the Logging interface and all of the implementors which made me like that idea less since it's something specific to just stackdriver. 

That said.. you gave me a better idea.. so I refactored it a bit so that the labels function is now on the stackdriver struct but we still do have a helper function to do the casting and the calling part that appengine can use

Reply:
Spoke with Jean-Claude f2f and going to go ahead and add it to the Logger interface so feel free to hold off on reviewing for now while I refactor those changes